### PR TITLE
Improve the rules.MTForceError.

### PR DESCRIPTION
**Pull request by teweitsai**
Improve the `rules.MTForceError` rule.

### CODE REVIEW

File: doc/version_history.rst [hunk 1]
```diff
@@ -6,6 +6,11 @@
 Version History
 ###############
 
+v1.20.1
+-------
+
+* Improve the `rules.MTForceError` rule.
```
Comment (wvreeven) [inline doc/version_history.rst:12]: v1.20.0 has not been released yet. Please add this to that version.

**Review by wvreeven — APPROVED** [review]
LGTM. See my comment before merging.

File: python/lsst/ts/watcher/rules/mt_force_error.py [hunk 2]
```diff
@@ -75,12 +86,33 @@ def get_schema(cls) -> dict[str, typing.Any]:
...
+                    default: 2
+                threshold_mtmount_acceleration:
+                    description: >-
+                        Threshold of the MTMount acceleration. The force error
+                        will not be checked if the acceleration is higher than
+                        this threshold. This is to avoid the wrong alarm from
+                        the overshoot of the control algorithm. The unit is
+                        deg/s^2.
+                    type: number
+                    default: 0.05
```
Comment (wvreeven) [inline python/lsst/ts/watcher/rules/mt_force_error.py:115]: You repeat all these default values in the configuration. Either remove them here or remove the default values from the ts_config_ocs config.

Comment (teweitsai) [reply python/lsst/ts/watcher/rules/mt_force_error.py:115]: Values are changed in the PR of ts_config_ocs. Thanks!

File: doc/version_history.rst (hunk 1 above)
Comment (teweitsai) [reply doc/version_history.rst]: Updated in the commit: [4c76002](https://github.com/lsst-ts/ts_watcher/pull/87/commits/4c76002fd1f2bc6a963e833840596a65117225c8). Thanks!

**Pull request merged**